lua/bufferline/render.lua

```diff
@@ -506,12 +500,17 @@ function render.enable()
       -- We're allowed to use relative paths for buffers iff there are no tabpages
       -- or windows with a local directory (:tcd and :lcd)
       local use_relative_file_paths = true
-      for tabnr,tabpage in ipairs(list_tabpages()) do
+
+      -- PERF: I didn't import `haslocaldir` since it is only used her, (just before calling `:mksession` and exiting vim)
+      local haslocaldir = vim.fn.haslocaldir
+
+      for tabnr, tabpage in ipairs(list_tabpages()) do
         if not use_relative_file_paths or haslocaldir(-1, tabnr) == 1 then
           use_relative_file_paths = false
           break
         end
-        for _,win in ipairs(tabpage_list_wins(tabpage)) do
+
+        for _, win in ipairs(tabpage_list_wins(tabpage)) do
```

Comment:
Normalizing style here too. In the rest of the project we do `for foo, bar in iterator do`.